src/Analyzers/CSharp/CodeFixes/UseIndexOrRangeOperator/CSharpUseRangeOperatorCodeFixProvider.cs

```diff
@@ -182,7 +184,7 @@ private static RangeExpressionSyntax CreateConstantRange(Result result)
             var constant1 = GetInt32Value(result.Op1);
             var constant2 = GetInt32Value(result.Op2);
 
-            var endExpr = (ExpressionSyntax)CSharpSyntaxGenerator.Instance.LiteralExpression(constant2 - constant1);
```

Comment:
`CSharpSyntaxGenerator` is an internal type not available in C# code style layer.